### PR TITLE
Refactor DAO classes and DB scripts

### PR DESCRIPTION
**Pull request by dilee**
**Purpose**
Refactoring DAO classes and DB scripts.

### CODE REVIEW

File: components/device-mgt/org.wso2.carbon.device.mgt.core/src/main/java/org/wso2/carbon/device/mgt/core/dao/impl/ApplicationMappingDAOImpl.java [hunk 1]
```diff
@@ -88,11 +88,11 @@ public int addApplicationMapping(int deviceId, int applicationId,
             }
             stmt.executeBatch();
 
-            rs = stmt.getGeneratedKeys();
-            while (rs.next()) {
-                mappingIds.add(rs.getInt(1));
-            }
-            return mappingIds;
+//            rs = stmt.getGeneratedKeys();
```
Comment (prabushi) [inline components/device-mgt/org.wso2.carbon.device.mgt.core/src/main/java/org/wso2/carbon/device/mgt/core/dao/impl/ApplicationMappingDAOImpl.java:91]: Remove commented code.

File: components/device-mgt/org.wso2.carbon.device.mgt.core/src/main/java/org/wso2/carbon/device/mgt/core/operation/mgt/dao/impl/PolicyOperationDAOImpl.java [hunk 2]
```diff
@@ -39,6 +39,8 @@
     public int addOperation(Operation operation) throws OperationManagementDAOException {
         int operationId;
         PreparedStatement stmt = null;
+        ByteArrayOutputStream bao = null;
+        ObjectOutputStream oos = null;
         try {
```
Comment (prabushi) [inline components/device-mgt/org.wso2.carbon.device.mgt.core/src/main/java/org/wso2/carbon/device/mgt/core/operation/mgt/dao/impl/PolicyOperationDAOImpl.java:44]: Cant we use try with resource here?

File: components/device-mgt/org.wso2.carbon.device.mgt.core/src/main/java/org/wso2/carbon/device/mgt/core/operation/mgt/dao/impl/ProfileOperationDAOImpl.java [hunk 3]
```diff
@@ -37,22 +37,46 @@
...
+        ObjectOutputStream oos = null;
+
         int operationId;
         try {
             operationId = super.addOperation(operation);
             operation.setCreatedTimeStamp(new Timestamp(new java.util.Date().getTime()).toString());
             operation.setId(operationId);
             operation.setEnabled(true);
-            ProfileOperation profileOp = (ProfileOperation) operation;
+            //ProfileOperation profileOp = (ProfileOperation) operation;
```
Comment (prabushi) [inline components/device-mgt/org.wso2.carbon.device.mgt.core/src/main/java/org/wso2/carbon/device/mgt/core/operation/mgt/dao/impl/ProfileOperationDAOImpl.java:49]: Remove commented line. And, can we change this to Java try with resource?

**Comment by prabushi** [comment]
Add a description.

File: components/device-mgt/org.wso2.carbon.device.mgt.core/src/main/java/org/wso2/carbon/device/mgt/core/dao/impl/ApplicationMappingDAOImpl.java (hunk 1 above)
Comment (dilee) [reply components/device-mgt/org.wso2.carbon.device.mgt.core/src/main/java/org/wso2/carbon/device/mgt/core/dao/impl/ApplicationMappingDAOImpl.java:91]: Fixed in https://github.com/wso2/carbon-device-mgt/commit/77c4c3485830cda733fff66fbf472f79d38df45c

File: components/device-mgt/org.wso2.carbon.device.mgt.core/src/main/java/org/wso2/carbon/device/mgt/core/operation/mgt/dao/impl/PolicyOperationDAOImpl.java (hunk 2 above)
Comment (dilee) [reply components/device-mgt/org.wso2.carbon.device.mgt.core/src/main/java/org/wso2/carbon/device/mgt/core/operation/mgt/dao/impl/PolicyOperationDAOImpl.java:44]: Fixed in https://github.com/wso2/carbon-device-mgt/pull/1275

File: components/device-mgt/org.wso2.carbon.device.mgt.core/src/main/java/org/wso2/carbon/device/mgt/core/operation/mgt/dao/impl/ProfileOperationDAOImpl.java (hunk 3 above)
Comment (dilee) [reply components/device-mgt/org.wso2.carbon.device.mgt.core/src/main/java/org/wso2/carbon/device/mgt/core/operation/mgt/dao/impl/ProfileOperationDAOImpl.java:49]: Fixed in https://github.com/wso2/carbon-device-mgt/pull/1275

**Pull request merged**